File: jct3appandroid/app/src/main/java/com/upn/trabajo3/appjct3/Entidades/Cita.java
package com.upn.trabajo3.appjct3.Entidades;

import java.sql.Date;

public class Cita {
	private int idcita;
	private Usuario usuario;
	private Historia historia;
	private Programacion programacion;
	private String fecha;
	public String getFecha() {
		return fecha;
	}
	public void setFecha(String fecha) {
		this.fecha = fecha;
	}
	public int getIdcita() {
		return idcita;
	}
	public void setIdcita(int idcita) {
		this.idcita = idcita;
	}
	public Usuario getUsuario() {
		return usuario;
	}
	public void setUsuario(Usuario usuario) {
		this.usuario = usuario;
	}
	public Historia getHistoria() {
		return historia;
	}
	public void setHistoria(Historia historia) {
		this.historia = historia;
	}
	public Programacion getProgramacion() {
		return programacion;
	}
	public void setProgramacion(Programacion programacion) {
		this.programacion = programacion;
	}

}

File: escritorio/Escritorio/Escritorio/Cita.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Escritorio
{
    class Cita
    {
        private int idcita;
     //   private Usuario usuario;
       // private Historia historia;
        private Programacion programacion;
        private String fecha;

        public Programacion Programacion
        {
            get
            {
                return programacion;
            }

            set
            {
                programacion = value;
            }
        }

        public string Fecha
        {
            get
            {
                return fecha;
            }

            set
            {
                fecha = value;
            }
        }

        public int Idcita
        {
            get
            {
                return idcita;
            }

            set
            {
                idcita = value;
            }
        }

        
    }
}
